spec/lib/vanagon/platform/windows_spec.rb

```diff
@@ -38,6 +59,114 @@
           expect(cur_plat._platform.target_user).to eq(plat[:target_user])
         end
       end
+
+      describe '#generate_msi_packaging_artifacts' do
+        before(:each) do
+          # Create Workdir and temp root directory
+          FileUtils.mkdir_p("#{workdir}/wix")
+          FileUtils.mkdir_p("#{VANAGON_ROOT}/resources/windows/wix")
+          FileUtils.mkdir_p("#{PROJ_ROOT}/resources/windows/wix")
+          @pwd = Dir.pwd
+          Dir.chdir(PROJ_ROOT)
+        end
+        after(:each) do
+          # Cleanup the complete work directory tree
+          FileUtils.rm_rf("#{PROJ_ROOT}")
+          FileUtils.rm_rf("#{VANAGON_ROOT}")
+          FileUtils.rm_rf("#{workdir}/wix")
+          Dir.chdir(@pwd)
+        end
+
+        it "Copies Wix File from product specific directory to output directory" do
+          # setup source directories and run artifact generation
+          FileUtils.cp("#{wixtestfiles}/file-1.wxs", "#{PROJ_ROOT}/resources/windows/wix/file-1.wxs")
+          cur_plat._platform.generate_msi_packaging_artifacts(workdir, project, binding)
```

Comment:
Need to pass ```plat[:projname]``` instead of ```project``` in all of these calls.

```cur_plat._platform.generate_msi_packaging_artifacts(workdir,plat[:projname], binding)```